diagnostic.md

```diff
@@ -150,7 +180,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+div one will be to the right of the screen
+div four will be the left, below the other divs
+the other divs will be on the left, lined from left to right
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5